apps/prairielearn/src/tests/sync/util.ts

```diff
@@ -4,7 +4,7 @@ import { assert } from 'chai';
 import stringify from 'fast-json-stable-stringify';
 import fs from 'fs-extra';
 import * as tmp from 'tmp-promise';
-import { z } from 'zod';
+import { type z } from 'zod';
```

Comment:
This is an interesting one, though it makes sense based on its use in this context.